wgpu-hal/Cargo.toml

```diff
@@ -109,7 +109,7 @@ features = ["wgsl-in"]
 
 [dev-dependencies]
 env_logger = "0.9"
-winit = "0.26"     # for "halmark" example
+winit = "~0.27.1"     # for "halmark" example
```

Comment:
This doesn't need to be a tilde dep, it can just be `0.27.1`

Reply:
Wouldn't we want to let it support 0.27.1 up till 0.28?

Reply:
Yeah, the standard dependency is an implicit carret dependency, so it would be the same as `>=0.27.1, <0.28.0`